src/Card/index.jsx

```diff
@@ -52,6 +52,9 @@ Card.defaultProps = {
   isClickable: false,
 };
 
+Card.Img = BaseCard.Img;
+Card.Title = BaseCard.Title;
+Card.Subtitle = BaseCard.Subtitle;
```

Comment:
Hm, so I don't think we actually want to expose these subcomponents any more, since card titles and subtitles should come through `Card.Header` now and `Card.Img` was largely replaced by `Card.ImageCap`.

Reply:
Yes, I understand, removed them